src/core/canonical_constructor.jl

```diff
@@ -1,40 +1,81 @@
 
-function _canonical_model_init(bus_count::Int64,
+function _container_spec(V::DataType, ax...; kwargs...)
+
+    if :parameters in keys(kwargs)
+        parameters = kwargs[:parameters]
+    else
+        parameters = true
+    end
+
+    # While JuMP fixes the isassigned problems
+    if parameters
```

Comment:
IMHO, this is a perfect place for using a ternary operator.

cont = parameters ? JuMP.Containers.DenseAxisArray{PGAE{V}}(undef, ax...) :
                    JuMP.Containers.DenseAxisArray{GAE{V}}(undef, ax...)
_remove_undef!(cont.data)
return cont

Reply:
This is a temporary code while a fix is added to JuMP to address the `isassigned` function. The code will be deleted later.